README.md

```diff
@@ -1,6 +1,59 @@
 # Express Wolox Logger
 
-ExpressJS logger that wraps [pino](https://github.com/pinojs/pino) with additional features like unique requestId and a requestMiddleware.
+ExpressJS logger that wraps [pino](https://github.com/pinojs/pino) with additional features like middlewares for unique request ids and automatic logging of request beginnings and endings.
+
+## Basic Usage
+```
+const { logger } = require('express-wolox-logger');
+
+logger.info('hello world');
+logger.error('something bad happened');
+```
+This will output:
+```
+[2019-06-14 17:35:13.772 +0000] INFO  (17439 on my-pc.local): hello world
+[2019-06-14 17:35:13.772 +0000] ERROR (17439 on my-pc.local): something bad happened
+```
+
+## Advanced Usage
+### Logs for request beginning and end
+We provide an ExpressJs middleware that automatically logs when a request starts and ends. Simply import it and use it like any other middleware:
+```
+const { expressMiddleware } = require('express-wolox-logger');
+
+app.use(expressMiddleware);
```

Comment:
be careful that in #2 this changed to a function that receives a logger and returns a middleware.